cohort_3/curriculum/workshops/portfolio/README.md

```diff
@@ -0,0 +1,271 @@
+- Click `Go to your dashboard`
+- Creating a workspace
+    - Click `Create a new workspace`
+    - Give it the name `hack-camp`
+    - Make sure the `Custom` type is selected
+    - Click `Create workspace`
+- Configuring your workspace
+    - Select the `Minimal Editor` preset.
+    - Change `Soft Tabs` to 2
+    - Turn on `Enable Auto-Save`
+    - At the very top of the window, click the little triangle on the bar to
+      show the Cloud9 editor options.
+
+      ![](img/c9_configuring_workspace_top_bar.png)
+
+    - Click `View` and then `Console` to open up the terminal
+    - Your workspace should now look like the following:
+
+      ![](img/c9_configured_workspace.png)
+
+    - Go ahead and close the welcome tab
+
+      ![](img/c9_closed_welcome_tab.png)
+
+    - Final step, paste the following command into the terminal on the bottom
+      and hit enter. This may take a few minutes to finish.
+
+            curl -sL https://git.io/vtbp6 | sudo dd of=/usr/local/bin/live_reload && sudo chmod +x /usr/local/bin/live_reload && live_reload
+
+    - When it's done, your workspace should look like the following
+
+      ![](img/c9_live_reload_installed.png)
+- Getting off the ground
+    - On the left file pane, right click and click `New Folder` to create a new
+      folder. Name it `portfolio``
+
+      ![](img/c9_create_portfolio_folder.gif)
+
+    - Create a new file inside of portfolio called `index.html` and open it
+
+      ![](img/c9_create_index_file.gif)
+
+    - Go ahead and type the following code into the open file (don't copy and
+      paste it! and make sure to indent properly!). Replace "Zaphod" with your
+      first name and "Zaphod Beeblebrox" with your first and last name.
+
+      ```html
+      <!DOCTYPE html>
+      <html>
+        <head>
+          <title>Zaphod's Portfolio</title>
+        </head>
+        <body>
+          <h1>Zaphod Beeblebrox</h1>
+        </body
+      </html>
+      ```
+
+      ![](img/c9_portfolio_template.gif)
+
+    - Let's see what our website looks like so far. **Do not** click the `Run`
+      button.
+
+      ![](img/c9_preview_template_website.gif)
+
+    - We can also open the website in its own tab.
+
+      ![](img/c9_preview_template_website_own_tab.gif)
+- Perfect. Let's get this website on the real internet.
+    - [GitHub](https://github.com/) is where millions of developers store their
+      code online. One of their services,
+      [GitHub Pages](https://pages.github.com/), will put a website online for
+      you. We'll be using GitHub pages to host our portfolio.
+    - Head over to https://github.com/ and create a free account.
+
+      ![](img/github_create_account.png)
+
+    - Once you have an account created, go ahead and create a new repository
+      with the name _username_.github.io (replacing _username_ with your GitHub
+      username). Make sure `Initialize this repository with a README` **is not**
+      checked.
+
```

Comment:
have breaks between sections? it was hard for me to know where i can pause and continue again later